src/util/mock-servers/service-testers/service-handlers/implementation/read-rows-handler.ts

```diff
@@ -0,0 +1,126 @@
+// Copyright 2022 Google LLC
```

Comment:
I should have caught this in the last review, since these servers (I believe) will only be used in testing, I'd be tempted to have them in `test/util`, rather than `src/util`. I think of `src` as being components of the library itself, vs., components used during testing.